packages/store/ts/config/v2/table.ts

```diff
@@ -159,13 +164,15 @@ export function resolveTable<input extends TableInput, scope extends Scope = Abi
   const label = input.label;
   const type = input.type ?? TABLE_DEFAULTS.type;
   const namespace = input.namespace ?? TABLE_DEFAULTS.namespace;
+  const namespaceLabel = input.namespaceLabel ?? input.namespace ?? TABLE_DEFAULTS.namespace;
```

Comment:
```suggestion
  const namespaceLabel = input.namespaceLabel ?? namespace;
```

for names, this is actually the other way around: user defines the name or it defaults to the label

I wonder if we should do the same here: allow specifying namespace label, default it if not set, then derive the actual namespace from either the input or the namespace label (and validate length?)